keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/constraints/CanaryConstraintEvaluator.kt

```diff
@@ -54,6 +55,7 @@ import org.springframework.stereotype.Component
  *    failureCancelsRunningRegions: false
  */
 @Component
+@ConditionalOnProperty("orca.enabled")
```

Comment:
Is this related to the bake retry issue?

Reply:
Ah, no. I should back this out